ports/cld3/unofficial-cld3Config.cmake

```
@@ -1,4 +1,4 @@
 include(CMakeFindDependencyMacro)
-find_dependency(Protobuf)
+find_dependency(protobuf)
```

```suggestion
find_dependency(protobuf CONFIG)
```

because `find_package(protobuf CONFIG REQUIRED)`

---

And literally `Protobuf`also for config mode, cf. https://github.com/microsoft/vcpkg/discussions/36934#discussioncomment-8598486
(vcpkg tool usage heuristics...)